<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Trace RGB Color:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Current RGB Values:</string>
    </property>
    <widget class="QLCDNumber" name="bColor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="rColor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="gColor">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="rLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Red:</string>
     </property>
    </widget>
    <widget class="QLabel" name="gLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Green:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Blue:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="colorPickerBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Use Color Picker</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>911</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Press to go back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startProcessing">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Press to go next</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
